chajda/tsquery/augments.py

```diff
@@ -2,7 +2,20 @@
 '''
 
 from chajda.tsvector import lemmatize, Config
-from chajda.tsquery import to_tsquery
+from chajda.tsquery.__init__ import to_tsquery
+
+
+from contextlib import contextmanager,redirect_stderr,redirect_stdout
```

Comment:
This chunk of code all comes from the stackoverflow question, right?

Whenever I copy/paste from stackoverflow, I always include a comment explaining why I've included the code here with a link to the original code.  These copy/pastes are often of esoteric concepts (as this one is), and so the comment can be extremely helpful in the future.